presentation/build.gradle

```diff
@@ -28,9 +28,17 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.2.0"
+    }
 }
 
 dependencies {
     implementation libs.bundles.compose
     implementation project(':core:designsystem')
+    implementation 'androidx.compose.ui:ui-tooling-preview:1.3.3'
```

Comment:
If you need add new implementation, add in the general gradle (consult with Irving)